Header header

uint8 danger
uint8 danger_assist 
uint8 route_outside
uint8 predict_outside
uint8 buffer
uint8 front_safe
uint8 revolute_safe

